### DISCUSSION TITLE
Is it safe to use Iranian CDN like Arvan Cloud? استفاده از سی‌دی‌ان ایرانی مثل ابر آروان امنه؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#117] [Q&A سوالات رایج] [answered]
آیا استفاده از سی‌دی‌ان ایرانی مثل ابر آروان امنه؟

**Comment by lymanjre** [comment]
من شخصا به سایتهای ایرانی که سرویس اینترنت ارائه میدن حالا cdn یا هر چیزی. اعتماد ندارم

**Comment by hiddify-com** [comment, marked as answer]
خیر به هیچ عنوان امن نیست؛ اما وقتی نت قطع میشه مجبوریم استفاده کنیم
بنابراین صرفا کانفیگ های با encryption در این شرایط پیشنهاد میشه

**Comment by kazem3d** [comment]
آیا پنل hiddify محدودتی در استفاده از cdn آروان دارد، چون تست کردم دیدم با cdn کلادفلر کار میکنه ولی با آروان خیر

**Reply by Sadegh-Kalami** [reply]
شما تونستی راه حلی ‍اوکی کنید؟

به خاطر port نیست ؟  سه روزه درگیرم و درست نشد که نشد.

**Reply by kazem3d** [reply]
مشکل برطرف شد و کار میکنه

**Reply by Sadegh-Kalami** [reply]
خب مشکل چی بود چون من نمیتونم هنووووز.
شاید تو راه‌حلی پیدا کردی من به هیچ وجه کار نمیکنه برام.
ممنون میشم با منم به اشتراک بزاری

**Comment by Leylahpn** [comment]
کار با ابزار های لینوکس چجوریه و زبان ها دیگه این وسط چی میشن؟
